backend/api/audio_generation.py

```diff
@@ -0,0 +1,46 @@
+from fastapi import FastAPI, HTTPException, APIRouter
+from pydantic import BaseModel
+from typing import Dict, Any
+import os
+import sys
+
+# Add project root to Python path
+ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(ROOT)
+
+try:
+    from utils.cerebras_connect import generate_audio
```

Comment:
`cerebras_connect.py` doesn't currently exist in utils there. I put it in the `backend/cerebras` folder - work in progress